views/next-steps.erb

```diff
@@ -19,32 +19,28 @@
 
     <div class="column-two-thirds">
       <h1 class="heading-large">
-        Start using GOV.UK PaaS
+        Access GOV.UK PaaS
       </h1>
 
       <p class="lede">
-        A lot of the tasks you’ll need to do to operate GOV.UK PaaS will be done from your computers
-        command line interface.
+        Installing the Cloud Foundry command line will let you operate GOV.UK PaaS from your computer
```

Comment:
Do we need a full-stop at the end of this sentence?

Reply:
I checked with Jess--yup!

Reply:
Nice. I'll merge this tomorrow morning.